usage/launcher/src/main/java/brooklyn/launcher/BrooklynWebServer.java

```diff
@@ -364,19 +367,28 @@ public synchronized void start() throws Exception {
 
             SslContextFactory sslContextFactory = new SslContextFactory();
 
-            if (keystorePath==null) keystorePath = managementContext.getConfig().getConfig(BrooklynWebConfig.KEYSTORE_URL);
+            // allow webconsole keystore & related properties to be set in brooklyn.properties
+            if (Strings.isNonBlank(keystorePath)) {
+                if (keystoreUrl==null) {
+                    log.warn("Deprecated 'keystorePath' used; callers should use 'keystoreUrl'");
+                    keystoreUrl = keystorePath;
+                } else if (!keystoreUrl.equals(keystorePath)) {
+                    log.warn("Deprecated 'keystorePath' supplied with different value than 'keystoreUrl', preferring the latter: "+
```

Comment:
If both `keystoreUrl` and `keystorePath` are given I think it would be better to throw rather than to log and ignore the latter. The warning message will be missed, it will not be obvious which property will be used when reading property files and we will end up with confused users.